Dockerfile

```diff
@@ -23,6 +23,6 @@ ADD . /home/jovyan/work
 
 WORKDIR /home/jovyan/work
 
-RUN pip install -q .
+RUN pip install -q '.[all]'
```

Comment:
I think this should fix https://github.com/unit8co/darts/issues/254

Reply:
I tested this locally, and it does not work. Instead `RUN pip install  .['all']` is working for me